lib/backend/k8s/conversion/constants.go

```diff
@@ -15,7 +15,9 @@
 package conversion
 
 const (
-	NamespaceLabelPrefix       = "pcns."
-	NamespaceProfileNamePrefix = "kns."
-	K8sNetworkPolicyNamePrefix = "knp.default."
+	NamespaceLabelPrefix            = "pcns."
+	NamespaceProfileNamePrefix      = "kns."
+	K8sNetworkPolicyNamePrefix      = "knp.default."
+	ServiceAccountLabelPrefix       = "pcsa/"
```

Comment:
just curious as to why these are `/` separated rather than `.`

Reply:
labels allow '.' in them :-( namespaces do not. and names also allow '.' in them.

Reply:
@caseydavenport  what's your take on using / here rather than . ?

@saumoh don't we return these names to the user - and AAIK you won't be able to have slashes in the names.  From the code I think you are joining the prefix + namespace + SAP name.  Since the namespace cannot have dots then I think using a dot separator will not result in any conflicts.

Reply:
@robbrockbank  i see what u mean. I can use '.' for both the ServiceAccountLabelPrefix and the ServiceAccountProfileNamePrefix.
I'll change the code in ProfileNameToServiceAccount to use strings.SplitN()